index.html

```diff
@@ -26,44 +31,92 @@ <h2 class="subtitle">
     <div class="field is-grouped has-addons" id="search-bar">
       <label class="label">Search Jobs</label>
       <div class="control">
+        <i class="fas fa-map-marker-alt"></i>
         <input
           class="input"
           type="text"
           placeholder="Search Job Title"
           id="enterInput"
         />
       </div>
-      <div>
-        <button class="button is-info" id="submit">Search</button>
-      </div>
-    </div>
-    <div class="field">
+
       <div class="control">
-        <textarea
-          class="textarea is-primary"
-          placeholder="Primary textarea"
-          id="text1"
-        ></textarea>
+        <span>
+          <input
+            class="input"
+            type="text"
+            placeholder="Location"
+            id="searchLoc"
+          />
+        </span>
       </div>
+      <div></div>
+      <button class="button is-info" id="submit">Search</button>
     </div>
+
+    <div class="contanier" id="main">
+      <aside id="browser">
+        <div class="columns is-vcentered">
+          <div class="column">
+            <p class="bd-notification is-primary" id="info1"></p>
+            <button class="button is-link" id="link" style="display: none;">
+              Apply
+            </button>
+            <p class="bd-notification is-primary" id="info2"></p>
+          </div>
+        </div>
+        <div class="columns is-vcentered">
+          <div class="column">
+            <h1 id="qualification" style="display: none;">
+              Qualification Summary:
+            </h1>
+            <br />
+            <p class="bd-notification is-primary" id="details"></p>
+          </div>
+        </div>
+      </aside>
+      <!-- 
     <div class="field">
-      <div class="control">
-        <textarea
-          class="textarea is-info"
-          placeholder="Info textarea"
-          id="text2"
-        ></textarea>
+      <div class="row" id="side-col">
+        <div class="column">hello1</div>
+        <div class="column">hello2</div>
       </div>
-    </div>
-    <div class="field">
-      <div class="control">
-        <textarea
-          class="textarea is-success"
-          placeholder="Success textarea"
-          id="text3"
-        ></textarea>
+    </div> -->
```

Comment:
Suganya, it looks like the code is commented out and isn't going to work. Did you mean to do this?